src/models/disassemblyoutput.cpp

```diff
@@ -176,6 +176,7 @@ DisassemblyOutput DisassemblyOutput::disassemble(const QString& objdump, const Q
 
     if (output.isEmpty()) {
         disassemblyOutput.errorMessage += QApplication::tr("Empty output of command %1").arg(objdump);
```

Comment:
this is now a string puzzle which makes localization hard/impossible for some languages

instead, write it like so:

```
disassemblyOutput.errorMessage += QApplication::tr("Empty output of command %1 %2").arg(objdump, arguments.join(QLatin1Char(' ')));
```

Reply:
Thanks for the review - that makes much more sense!